core/sql/src/main/java/org/trafodion/sql/HBaseClient.java

```diff
@@ -648,6 +648,8 @@ public boolean truncate(String tblName, boolean preserveSplits, long transID)
            }
            else {
               TableName tableName = TableName.valueOf(tblName);
+              if (admin.isTableEnabled(tableName))
+                  admin.disableTable(tableName);
```

Comment:
HBase truncate api requires table to be disabled.
truncate api enables the table after truncation.